sqldb/sqlite.go

```diff
@@ -0,0 +1,112 @@
+package sqldb
```

Comment:
question: would it be reasonable to only include this file when building for example with the kvdb_sqlite package, this came to my mind because later in the `test` files you mark the with specific tags like `test_db_postgres` or `test_db_sqlite` so was wondering why in the test code but not here?

Reply:
+1